### PR TITLE
RES-83 Add NAB to NuPIC regression tests (v. 3)

### PR DESCRIPTION
**Pull request by SaganBolliger**
https://jira.numenta.com/browse/RES-83
Third attempt to test the travis code run after #25, #26. 

Fixed bad path to NAB by setting NAB environment variable in travis and using it in the test.

@rhyolight Can you merge this to test again?

### CODE REVIEW

**Comment by rhyolight** [comment]
https://travis-ci.org/numenta/nupic.regression/builds/85444170

**Comment by SaganBolliger** [comment]
Looks like this one is good.

**Comment by rhyolight** [comment]
:+1:

File: tests/anomaly/NAB/NAB_anomaly_test.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import unittest2 as unittest
-import json
+import json, os
```
Comment (scottpurdy) [inline tests/anomaly/NAB/NAB_anomaly_test.py]: Don't import on same line

File: tests/anomaly/NAB/NAB_anomaly_test.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ class NABAnomalyTest(unittest.TestCase):
 
   def testNABScoresBenchmark(self):
     """Test that NAB on nupic produces expected scores."""
-
+
```
Comment (scottpurdy) [inline tests/anomaly/NAB/NAB_anomaly_test.py]: remove trailing whitepace on this line

**Pull request merged**